tests/templates/kuttl/kerberos/01-install-kdc.yaml

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
That's an option too, but it also requires us to do the same thing for deploying the consuming pod...

Reply:
That's correct. My aim would not be to make this as easy as possible, but to get some sort of established (universal) pattern, so that the tests are consistent and not every person comes up with a different solution.
Added an arch meeting entry, maybe we can decide on a "standard"

Reply:
Do we want to hold this up or move it to a separate ticket to resolve after the arch meeting?

Reply:
I'm fine with both ;) Feel free to resolve